configs/tsimx9_debian_12_headless_defconfig

```diff
@@ -2,11 +2,17 @@ CONFIG_DS_ARCH_AARCH64=y
 CONFIG_DS_DISTRO_DEBIAN_12=y
 CONFIG_DS_PACKAGELIST="debian_12_headless.txt"
 CONFIG_DS_KERNEL_PROVIDER_GIT_URL="git@github.com:embeddedTS/prototype-linux-imx93.git"
-CONFIG_DS_KERNEL_PROVIDER_GIT_VERSION="bf62281ae2c421ccb1c5425609fc0e38ab3ec913"
+CONFIG_DS_KERNEL_PROVIDER_GIT_VERSION="ee246c7c97f1c75ccbdf5e1617cdccd53d173bb7"
 CONFIG_DS_KERNEL_DEFCONFIG="ts9370_defconfig"
 CONFIG_DS_KERNEL_INSTALL_DEVICETREE_FILESYSTEM="freescale/imx93-ts4300-16 freescale/imx93-ts4300-16-ts-rd-mipi2dp freescale/imx93-ts4300-16-ts-rd-mipi2dp-ti freescale/imx93-ts4300 freescale/imx93-ts9370 freescale/imx93-ts9390"
 CONFIG_DS_COMPONENT_TSSUPERVISORUPDATE=y
 CONFIG_DS_COMPONENT_TS_UBOOT_CONFIGS=y
+CONFIG_DS_MODULE_MWIFIEX=y
+CONFIG_DS_MODULE_MWIFIEX_GIT_VERSION="lf-6.6.52_2.2.0"
+CONFIG_DS_COMPONENT_IMX_FIRMWARE=y
+CONFIG_DS_COMPONENT_IMX_FIRMWARE_GIT_VERSION="lf-6.6.52_2.2.0"
+CONFIG_DS_COMPONENT_MURATA_NXP_LINUX_CALIBRATION=y
+CONFIG_DS_COMPONENT_MURATA_NXP_LINUX_CALIBRATION_GIT_VERSION="imx-6-6-23"
```

Comment:
I'm going to point at the Buildroot paradigm and say that the actual version belongs in the package files (e.g. `fetch.sh`) rather than the defconfig unless there is a very specific reason that each of these, `CONFIG_DS_MODULE_MWIFIEX_GIT_VERSION`, `CONFIG_DS_COMPONENT_IMX_FIRMWARE_GIT_VERSION`, and `CONFIG_DS_COMPONENT_MURATA_NXP_LINUX_CALIBRATION_GIT_VERSION` would ever need to change independently of each other.

The defconfig should drive whether or not the feature/package is included, and the actual feature/package should always use the latest unless there is some reason that does not make sense to do.